newscoop/src/Newscoop/GimmeBundle/Controller/ArticlesListController.php

```diff
@@ -141,17 +144,13 @@ public function getPlaylistsArticlesAction(Request $request, $id)
         }
 
         $playlistArticles = $em->getRepository('Newscoop\Entity\Playlist')
-            ->articles($playlist, null, false, null, null, $onlyPublished, true)->getResult();
+            ->articles($playlist, array(), true, null, null, $onlyPublished, true)->getResult();
 
-        $articlesIds = array();
+        $articles = array();
         foreach ($playlistArticles as $playlistArticle) {
-            $articlesIds[] = $playlistArticle['articleId'];
+            $articles[] = $playlistArticle->getArticle();
```

Comment:
yeah but take a look at: https://github.com/takeit/Newscoop/blob/playlist/newscoop/library/Newscoop/Entity/Repository/PlaylistRepository.php#L64

I rewrote this method from the native query to query builder, so it returns only objects of PlaylistArticle. Before it was returning mixed types of objects (Article and PlaylistArticle types)